<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopUpModel</class>
 <widget class="QWidget" name="PopUpModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;
border-radius:10px</string>
  </property>
  <widget class="QWidget" name="container_pop_up" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="pop_up_icon">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/img/img/success_icon.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="title_container_pop_up">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 114px;
height: 30px;
left: 576px;
top: 263px;

font-family: 'Inter';
font-style: normal;
font-weight: 400;
font-size: 25px;
line-height: 30px;
text-align: center;
color: #989797;</string>
    </property>
    <property name="text">
     <string>Success !</string>
    </property>
   </widget>
   <widget class="QLabel" name="description_pop_up">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>160</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">position: absolute;
width: 281px;
height: 32px;
left: 496px;
top: 311px;


font-style: normal;
font-weight: 400;
font-size: 13px;
line-height: 16px;
text-align: center;

color: #000000;</string>
    </property>
    <property name="text">
     <string>The product have been added to this staging 
                    table successfully</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
